integration_tests/dbt_project.yml

```diff
@@ -23,7 +30,18 @@ vars:
     salesforce_product_2_identifier: "sf_product_2_data"
     salesforce_order_identifier: "sf_order_data"
     salesforce_opportunity_line_item_identifier: "sf_opportunity_line_item_data"
-    salesforce_schema: salesforce_integrations_tests_1
+
+    ##history mode identifiers
+    salesforce_account_history_identifier: "sf_account_history_data"
+    salesforce_contact_history_identifier: "sf_contact_history_data"
+    salesforce_event_history_identifier: "sf_event_history_data"
```

Comment:
Same comment about removing seeds and identifiers for history models that are no longer used.